autogen/coding/commandline_code_executor.py

```diff
@@ -82,34 +115,24 @@ def execute_code_blocks(self, code_blocks: List[CodeBlock]) -> CodeResult:
                 flush=True,
             )
             if lang in ["bash", "shell", "sh"]:
-                exitcode, logs, image = execute_code(
-                    code=code,
-                    lang=lang,
-                    timeout=self.timeout,
-                    work_dir=self.work_dir,
-                    filename=self.filename,
-                    use_docker=self._get_use_docker_for_code_utils(),
-                )
+                exitcode, logs, image = self._execute_code(code, lang)
+                # exitcode, logs, image = execute_code(
+                #     code=code,
+                #     lang=lang,
+                #     timeout=self.timeout,
+                #     work_dir=self.work_dir,
+                #     filename=self.filename,
+                #     use_docker=self._get_use_docker_for_code_utils(),
```

Comment:
this was wrong because `None` can be returned by `_get_use_docker_for_code_utils`

Reply:
fixed `_get_use_docker_for_code_utils `